src/diem/identifier/__init__.py

```diff
@@ -57,16 +57,22 @@ def __init__(
         sub_address: typing.Optional[bytes],
         currency_code: str,
         amount: int,
+        hrp: str,
     ) -> None:
         self.account_address = account_address
         self.sub_address = sub_address
         self.currency_code = currency_code
         self.amount = amount
+        self.hrp = hrp
 
     @property
     def account_address_bytes(self) -> bytes:
         return self.account_address.to_bytes()
 
+    @property
+    def account_id(self) -> str:
```

Comment:
Are we calling the bech32 address account IDs?

Reply:
see https://dip.diem.com/dip-5/#account-identifiers